lib/methods/import.js

```diff
@@ -751,8 +751,8 @@ module.exports = self => {
         // recomputeAllDocReferences
         //
         // Perform any crops not found in the existing attachment
-        for (const crop of [ ...(attachment.crops || []), ...(existing.crops || []) ]) {
-          if (!existing.crops.find(c => JSON.stringify(c) === JSON.stringify(crop))) {
+        for (const crop of attachment.crops || []) {
+          if (!(existing.crops || []).find(c => JSON.stringify(c) === JSON.stringify(crop))) {
```

Comment:
This is where the money's at: the crops property is not a valid array until the first one is added, so we have to guard against that to prevent the bug